help/destinations/catalog/personalization/algolia.md

```diff
@@ -0,0 +1,145 @@
+
+## Use cases {#use-cases}
+
+To help you better understand how and when you should use the [!DNL Algolia] destination, here are sample use cases that Adobe Experience Platform customers can solve by using this destination.
+
+### Personalization consistency {#personalization-consistency}
+
+Use this destination connector to deliver a consistent personalization across your site from home page to search.  
+
+For example, as a marketer, you might want to build rich audiences in Adobe Experience Platform from multiple user data sources, including Algolia. You can use the [!DNL Algolia] connector to share the audiences for targeting strategies, leading to a boost in campaign personalization and conversion.
+
+To implement this use case, you must use both the [[!DNL Algolia] source](/help/sources/connectors/data-partners/algolia-user-profiles.md) and destination connectors.
+
+You would start by importing your existing [!DNL Algolia] user profiles into Adobe Experience Platform Real-Time CDP and other sources to begin creating rich audiences with the source connector. Marketers would create audiences using the profile data that can be sent to Algolia for search and recommendation personalization.
+
+Then, use the corresponding [[!DNL Algolia User Profiles]](/help/sources/connectors/data-partners/algolia-user-profiles.md) source connector to ingest and augment customer profiles back into Real-Time CDP.
+
+## Prerequisites {#prerequisites}
+
+>[!IMPORTANT]
+>
+>* To connect to the destination, you need the **[!UICONTROL View Destinations]** and **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+## Supported identities {#supported-identities}
+
+[!DNL Algolia] supports the activation of identities described in the table below. Learn more about [identities](https://experienceleague.adobe.com/en/docs/experience-platform/identity/features/namespaces).
+
+| Target Identity | Description | Considerations |
+|---------|---------|----------|
+| AlgoliaUserToken | Algolia User Token | Select this identity to map to the user profile in Algolia to associate the [!DNL Adobe Experience Platform] audience ids. |
```

Comment:
Is the `AlgoliaUserToken` the only target identity that is supported by this destination?